index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <link rel="icon" href="Tesla.svg">
+        <link rel="stylesheet" href="style.css" />
+        <title>Elon Musk Tribute</title>
+    </head>
+    
+    <body class="main-container">
+        <div id="main">
+            <h1 id="title" class="heading">Elon Musk</h1>
+            <div class="img-div" id="img-div">
+                <p id="img-caption" class="description">Below is an image of Elon Musk in front of a Tesla Sign</p>
+        <img id="image" class="image" src="musk-tesla.jpg" alt="Picture of Elon Musk"
+          width="800px"/>
```

Comment:
Please ensure you check the code to look for errors and bugs and leave nice comments containing change suggestions for each.
Examples of issues to look out for when performing the review:

- Lack of [semantic HTML tags use](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md#:~:text=Use%20the%20appropriate%20tags%20for%20each%20element%20(e.g.%2C%20links%2C%20titles%2C%20etc.)%20and%20use%20HTML5%20semantic%20tags%20(e.g.%2C%20header%2C%20nav)%20over%20divs.).
- Not applying [HTML & CSS best practices correctly](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)
- Bugs missed the first time the project was built.
- And more.

Reply:
Please always includes an explanation, a "why", for each requested change. For example:
* This is a well-explained request: *Why don't you use an article tag for the .first-article element instead of a div, that would improve the HTML structure and make the page more accessible*